src/routes/+page.server.ts

```diff
@@ -149,14 +150,19 @@ export async function load() {
 	});
 
 	const latestVoteEvents: ComponentProps<VoteCard>[] = await Promise.all(
-		voteEvents.map(async (voteEvent) => ({
-			...voteEvent,
-			date: voteEvent.start_date,
-			votesByGroup: groupVotesByAffiliation(await queryPoliticiansVote(voteEvent)).map((aff) => ({
+		voteEvents.map(async (voteEvent) => {
+			const groupedVotes = groupVotesByAffiliation(await queryPoliticiansVote(voteEvent));
+			const groups = groupedVotes.map((aff) => ({
```

Comment:
Same as previous comment, this variable name was already used from import statement.